<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Install_local_snap_dialog</class>
 <widget class="QDialog" name="Install_local_snap_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snap Package Installer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="snap_path">
       <property name="placeholderText">
        <string>Select a .snap package</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="choose_snap">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="devmode">
       <property name="text">
        <string>install with --devmode flag</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_snap">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="install_button">
       <property name="text">
        <string>Install</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QTextEdit" name="terminal_output">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#0D0D0D;
color: rgb(128, 89, 120);
background-image: url(:/images/images/grid.png);
background-attachment: fixed;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="list">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#0D0D0D;
color: rgb(128, 89, 120);
background-image: url(:/images/images/grid.png);
background-attachment: fixed;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_dialog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
